.github/workflows/integration-test.yml

```
@@ -1170,3 +1170,23 @@ jobs:
           timeout_minutes: 8
           retry_wait_seconds: 5
           command: cd terraform/stress && terraform destroy --auto-approve
+          
+  EKSEndToEndTest:
+    name: "E2E EKS Test"
```

Can we reference AppSignals to differentiate this from our other integ tests?
```suggestion
    name: "AppSignals E2E EKS Test"
```